bot.lua

```diff
@@ -4502,6 +4503,13 @@ commands["lua"] = {
 
 				assert((msg.channel.id ~= channels["commu"] and msg.channel.id ~= channels["modules"]), "Message deletion denied.")
 
+				if not message.guild then -- dms
+					assert(msg.author.id == client.user.id, "You can only delete Modulo's messages in DMs.")
```

Comment:
This is not necessary, bot can never delete others' messages....

Reply:
It can in your server. Either way, it's better to have this message.

Reply:
Only in the guild, not in DMs. Remove it.

You have to change all `not message.guild` to the type solution